db/migrate/20220319150610_create_organizations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrganizations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :organizations do |t|
+      t.string :name, NULL: false
```

Comment:
Could you change all the words `NULL`to lower case?

Reply:
Sure, I'll do it.